src/DetailsView/components/tab-stops/requirements.ts

```diff
@@ -8,42 +8,54 @@ import {
     TabStopRequirementInfo,
 } from 'types/tab-stop-requirement-info';
 
-export const requirements: TabStopRequirementInfo = {
-    'keyboard-navigation': {
-        name: 'Keyboard navigation',
-        description: 'All interactive elements can be reached using the Tab and arrow keys. ',
-        guidance: [link.WCAG_2_1_1],
-    },
-    'keyboard-traps': {
-        name: 'Keyboard traps',
-        description:
-            'There are no interactive elements that “trap” input focus and prevent navigating away.',
-        guidance: [link.WCAG_2_1_2],
-    },
-    'focus-indicator': {
-        name: 'Focus indicator',
-        description:
-            'All interactive elements give a visible indication when they have input focus.',
-        guidance: [link.WCAG_2_4_7],
-    },
-    'tab-order': {
-        name: 'Tab order',
-        description:
-            "The tab order is consistent with the logical order that's communicated visually.",
-        guidance: [link.WCAG_2_4_3],
-    },
-    'input-focus': {
-        name: 'Input focus',
-        description: 'Input focus does not move unexpectedly without the user initiating it.',
-        guidance: [link.WCAG_2_3_1],
-    },
+export const requirements: (boolean) => TabStopRequirementInfo = (
+    includeAutomationInfo: boolean,
+) => {
+    return {
+        'keyboard-navigation': {
+            name: 'Keyboard navigation',
+            description: `All interactive elements can be reached using the Tab and arrow keys.${
+                includeAutomationInfo ? ' (Partially automated)' : ''
```

Comment:
Would it make sense to put "(Partially automated)" and "(Manual)" in variables? There are only 5 requirements so it may not be necessary. Wondering what your thoughts are